plugins/uefi-capsule/meson.build

```diff
@@ -10,6 +10,8 @@ endif
 install_data(['uefi-capsule.quirk'],
   install_dir: join_paths(datadir, 'fwupd', 'quirks.d'))
 
+esrt_impl_source = 'fu-uefi-esrt-@0@.c'.format(host_machine.system())
```

Comment:
what happens if host_machine.system() == windows? I think we need to do the `if`, `elif`, `else` -> `error()` pattern here I'm afraid.

Reply:
Or add dummy code for Windows for now.  I don't think it's going to be impossible to get the same info on Windows....